cli/core/install.go

```diff
@@ -54,6 +55,7 @@ func initInstallCommand() *cobra.Command {
 		},
 	}
 	postInstallFlags.AddToCommand(installCommand)
+	installCommand.Flags().BoolVar(&noOverwrite, "no-overwrite", false, tr("Do not overwrite existing platforms (during upgrade or downgrade)."))
```

Comment:
```suggestion
	installCommand.Flags().BoolVar(&noOverwrite, "no-overwrite", false, tr("Do not overwrite already installed platforms."))
```

Align flag description with `lib install --no-overwrite`.

It would be worth considering moving this shared flag to the `github.com/arduino/arduino-cli/cli/arguments` package. I don't think the required switch to a generally applicable description (e.g., "Do not overwrite existing installations.") would be harmful in this case.